mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -284,6 +268,7 @@ pub enum StorageSettings {
 
 #[derive(Debug, Clone, PartialEq, Deserialize)]
 pub struct MemorySettings {
+    #[serde(deserialize_with = "deserialize_nonzerouusize")]
     max_size: NonZeroUsize,
```

Comment:
Let's make it HumanSize instead?

Reply:
While we are here, are there any other forms of number format that we want to support? This will handle string and int format, not sure there are others? Like 2Kb or something?

Reply:
It looks like `HumanSize` will not work as it is right now, since it is backed by `usize`, where we need `u64`. We can add another version of `HumanSize(u64)` but I would rather do it in a separate PR.